scripts/synthetic_59_target_pair_soak.py

```
@@ -0,0 +1,656 @@
+    url: str,
+    token: str | None = None,
+    body: bytes | None = None,
+    content_type: str | None = None,
+    timeout: float = 600.0,
+) -> dict:
+    headers: dict[str, str] = {}
+    if token:
+        headers["Authorization"] = f"Bearer {token}"
+    if content_type:
+        headers["Content-Type"] = content_type
+    req = urllib.request.Request(url, data=body, headers=headers, method=method)
```

Ruff (0.16.1)

[error] Audit URL open for permitted schemes. Allowing use of `file:` or custom schemes is often unexpected.

(S310)